### PR TITLE
Updating energy source sponsor

### PR DESCRIPTION
**Pull request by melanierogan**
Adding the BP sponsor logo to the energy source email

### CODE REVIEW

File: templates/default-split-comment-sponsor.html [hunk 1]
```diff
@@ -137,6 +137,18 @@
                         <td>
 
                         {{header}}
+                            
+                       <table cellspacing="0" cellpadding="3" border="0" width="100%" align="center">
+                          <tbody>
+                               <tr style="background:white;">
+                                    <td align="center" class="header" valign="top" style="text-align: center; padding: 15px 0px;">
+                                      <a href="https://bs.serving-sys.com/serving/adServer.bs?cn=trd&mc=click&pli=27298756&PluID=0&ord=[timestamp]">
+                                        <img src="https://www.ft.com/__origami/service/image/v2/images/raw/https://email-platform-ftcom-manual.s3.amazonaws.com/images/5c489c7a2735aa001cc3e0f7.jpeg?source=ft-email-manual" align="center" class="bigLogo" alt="Sponsored by BP">
```
Comment (keirog) [inline templates/default-split-comment-sponsor.html]: Minor: ~`Sponsored by BP`~ `In partnership with BP` ?

File: templates/default-split-comment-sponsor.html [hunk 2]
```diff
@@ -137,6 +137,18 @@
                         <td>
 
                         {{header}}
+                            
+                       <table cellspacing="0" cellpadding="3" border="0" width="100%" align="center">
+                          <tbody>
+                               <tr style="background:white;">
+                                    <td align="center" class="header" valign="top" style="text-align: center; padding: 15px 0px;">
+                                      <a href="https://bs.serving-sys.com/serving/adServer.bs?cn=trd&mc=click&pli=27298756&PluID=0&ord=[timestamp]">
```
Comment (keirog) [inline templates/default-split-comment-sponsor.html]: Just logging here that we talked IRL about the `[timestamp]` bit (where we’re just going with what we were provided with, in lieu of having any way to validate that it’s correct)

Comment (AbdullahAli) [inline templates/default-split-comment-sponsor.html]: I was not able to open that link - is this expected?

Comment (keirog) [reply templates/default-split-comment-sponsor.html]: Works for me ¯\_(ツ)_/¯

Comment (AbdullahAli) [reply templates/default-split-comment-sponsor.html]: Yeh, odd, it does not load on my laptop but does on my phone

**Pull request merged**